docs/source/getting_started/index.rst

```diff
@@ -4,16 +4,134 @@
+   Ansys license, install DPF Server with:
+
+.. card::
+
+    #. Download the ansys_dpf_server_win_v2023.2.pre0.zip or ansys_dpf_server_lin_v2023.2.pre0.zip
+       file as appropriate.
+    #. Unzip the package and go to the root folder of the unzipped package
+       (ansys_dpf_server_win_v2023.2.pre0 or ansys_dpf_server_lin_v2023.2.pre0).
+    #. In a Python environment, run the following command:
+
+    .. code::
+
+        pip install -e .
+
+    #. DPF Server is protected using license terms specified in the DPFUserLicensingAgreement.txt
+       file that can be found on the Ansys Customer Portal. To accept the DPF User Licensing
+       Agreement terms, the following environment flag must be set:
+
+    .. code::
+
+        ANSYS_DPF_ACCEPT_LA=Y
+
+
+
+For other ways of using the Server (without pip install, user **Docker Containers**...), see
+:ref:`ref_getting_started_with_dpf_server`.
+
+
+Use PyDPF-Core
+--------------
+
+In the same Python environment, run the following command:
+
+.. code:: python
+
+    >>> from ansys.dpf import core as dpf
+    >>> from ansys.dpf.core import examples
+    >>> model = dpf.Model(examples.download_crankshaft())
+    >>> print(model)
+
+
+.. rst-class:: sphx-glr-script-out
+
+ .. code-block:: none
+
+    DPF Model
+    ------------------------------
+    Static analysis
+    Unit system: MKS: m, kg, N, s, V, A, degC
+    Physics Type: Mechanical
+    Available results:
+         -  displacement: Nodal Displacement
+         -  velocity: Nodal Velocity
+         -  acceleration: Nodal Acceleration
+         -  reaction_force: Nodal Force
+         -  stress: ElementalNodal Stress
+         -  elemental_volume: Elemental Volume
+         -  stiffness_matrix_energy: Elemental Energy-stiffness matrix
+         -  artificial_hourglass_energy: Elemental Hourglass Energy
+         -  thermal_dissipation_energy: Elemental thermal dissipation energy
+         -  kinetic_energy: Elemental Kinetic Energy
+         -  co_energy: Elemental co-energy
+         -  incremental_energy: Elemental incremental energy
+         -  elastic_strain: ElementalNodal Strain
+         -  structural_temperature: ElementalNodal Temperature
+    ------------------------------
+    DPF  Meshed Region:
+      69762 nodes
+      39315 elements
+      Unit: m
+      With solid (3D) elements
+    ------------------------------
+    DPF  Time/Freq Support:
+      Number of sets: 3
+    Cumulative     Time (s)       LoadStep       Substep
+    1              1.000000       1              1
+    2              2.000000       1              2
+    3              3.000000       1              3
+
+
+
+.. code:: python
+
+    >>> over_time_disp = model.results.displacement().eval()
+    >>> over_time_disp[0].plot()
+
+
+.. figure:: ../images/plotting/crankshaft_disp.png
+
 
 .. toctree::
-   :maxdepth: 3
+   :hidden:
    
    compatibility
    install
    dependencies
-   docker
```

Comment:
@cbellot000 maybe we should delete the docker.rst file ?

Reply:
@anslpa I removed it from the doc it will not appear, but we still have the file